Automattic/wp-calypso
# Jetpack Settings: Consolidate info link container styles

Part of #9171.

This PR cleans up some CSS that we duplicated in the development process of Jetpack Settings in Calypso. Previously this CSS was necessary to align the info icon to the right:
![screen shot 2017-02-27 at 16 39 30](https://cloud.githubusercontent.com/assets/8436925/23365361/8245539e-fd0b-11e6-9fbd-eb42d6d60577.png)

This PR cleans it up by consolidating all these to use the same class, and removes all obsolete duplicate declarations.

To test:
* Go to `/settings/writing/$site` and `/settings/discussion/$site` for a Jetpack site
* Verify there are no visual changes with the info icon in all any Jetpack module settings cards.
* Verify there are no remaining duplications of this CSS declaration within any Jetpack Settings cards.
